StRoot/StFttClusterMaker/StFttClusterMaker.h

```diff
@@ -47,12 +47,12 @@ class StFttClusterMaker: public StMaker {
 
     StEvent*             mEvent;
     StFttCollection*     mFttCollection;
-    int                  mRunYear;
-    bool                 mDebug;
+    Int_t                mRunYear;
+    Bool_t               mDebug;
     StFttDb*             mFttDb;
 
 
-    ClassDef( StFttClusterMaker, 1 )
+    ClassDef( StFttClusterMaker, 2 )
```

Comment:
I do not see a change to the memory layout here.  So ClassDef does not need to be incremented.  And since the maker will not be streamed, you can use ClassDef( StFttClusterMaker, 0).

Reply:
See comment below, all changes related to FTT reverted since they should be handled in PR #569 .
Done with https://github.com/star-bnl/star-sw/pull/571/commits/6088f974a30af0141f460b3fdd3183ae4f7c66ec